lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,37 @@
 import React from 'react';
 import Tab from './Tab';
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
 const Tabs = props => {
   return (
     <div className="tabs">
       <div className="topics">
-        <span className="title">TRENDING TOPICS:</span>
-        {/* map over the tabs provided on your props, create a new Tab component for each one.
-            give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/}
+        <Title>TRENDING TOPICS:</Title>
+        {props.tabs.map(item => (
+          <Tab
+            tab={item}
+            selectTabHandler={props.selectTabHandler(item)}
```

Comment:
if you make the change I recommended for your changeSelected method, then you would do this instead:
`selectTabHandler={() => props.selectTabHandler(item)}`